From 5f9ec46cf3d496db354987368a6ecefc8f2a3358 Mon Sep 17 00:00:00 2001
From: Robert Marko <robert.marko@sartura.hr>
Date: Mon, 2 Nov 2020 23:22:24 +0100
Subject: [PATCH] arm64: dts: ipq8074: disable QUSB2 PHY by default

One QUSB2 HS USB PHY was left enabled by default, most likely by
a mistake in copy/paste.
Since not all boards actually have the USB connected, disable that
PHY as well by default in line with others.

Signed-off-by: Robert Marko <robert.marko@sartura.hr>
---
 arch/arm64/boot/dts/qcom/ipq8074.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/qcom/ipq8074.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq8074.dtsi
@@ -194,6 +194,7 @@
 			clock-names = "cfg_ahb", "ref";
 
 			resets = <&gcc GCC_QUSB2_0_PHY_BCR>;
+			status = "disabled";
 		};
 
 		pcie_phy0: phy@86000 {
